recipe/meta.yaml

```
@@ -23,28 +22,31 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - llvm-openmp    # [osx]
   host:
     # Host dependencies are pinned to last build
     - python
     - cython 0.29.33
```

```suggestion
    - cython 0.29
```
This will be better for easier rebuilds